src/renderer/component/subHeader/index.js

```diff
@@ -1,6 +1,5 @@
-import React from 'react';
 import { connect } from 'react-redux';
```

Comment:
`component/subHeader` can just be deleted. Guess I didn't delete it, nothing is using it.